processing/src/test/java/io/druid/query/lookup/LookupReferencesManagerTest.java

```diff
@@ -169,4 +180,75 @@ public void testputAllExceptionWhenStartFail()
     );
     lookupReferencesManager.put(extractorImmutableMap);
   }
+
+  @Test
+  public void testBootstrapFromFile() throws IOException
```

Comment:
Can you add a test showing what happens on corrupt input file?

If needed you should be able to mock ObjectMapper to throw IOException on the deserialize call